rotkehlchen/exchanges/coinbase.py

```diff
@@ -384,6 +330,11 @@ def _api_query(
             except requests.exceptions.RequestException as e:
                 raise RemoteError(f'Coinbase API request failed due to {e!s}') from e
 
+            if response.status_code in (401, 429) and had_4xx is False:
+                had_4xx = True
+                gevent.sleep(.5)
+                continue  # do a single retry since they don't have info on retries
```

Comment:
This I put to try and combat these random 401 authentication error warnings. They also mention them in the forums but no idea why it happens. Seems to be on the coinbase side.